app/commands/donations/paypal/subscription/handle_recurring_payment.rb

```diff
@@ -0,0 +1,23 @@
+# Handle a Paypal IPN event with 'txn_type' = 'recurring_payment'
+class Donations::Paypal::Subscription::HandleRecurringPayment
```

Comment:
When a new subscription is created, the `recurring_payment_profile_created` event is sent first (which will create the subscription), followed by this event for the payment